src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -8181,52 +8181,174 @@ public class Test : ITest
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
-        public async Task GenericInterfaceNotNull1()
+        public async Task TestAllowNullAttributeOnNonNullableReferenceType()
```

Comment:
The name here is a bit confusing as it's kinda testing both AllowNull and MaybeNull at the same time...